### DISCUSSION TITLE
Enforcing IntelliJ IDEA "Support screen readers" to be on

### DISCUSSION THREAD

**Discussion by akkadaya** [aws/codewhisperer-command-line-discussions#166] [Known bugs]
Related to: https://youtrack.jetbrains.com/issue/IJPL-149202

"Amazon Q for command line" IntelliJ plugin is enforcing "Support screen readers" to be on.
This is causing "Tooltips on mouse hover" to be disabled in IntelliJ IDEA.

**Comment by vbtwo31984** [comment]
This also disables ctrl-tab switching to recently used files, a shortcut that I use a lot.
